cypress/support/config.js

```diff
@@ -18,9 +20,11 @@ const config = {
     assetUrl: 'https://news.test.files.bbci.co.uk/include/articles/public',
     assetOrigin: 'https://news.test.files.bbci.co.uk',
     assets: {
-      news: 'c0g992jmmkko',
-      newsThreeSubheadlines: 'c6v11qzyv8po',
+      news: 'c6v11qzyv8po',
       persian: 'c4vlle3q337o',
+    },
+    specialAssets: {
```

Comment:
Whilst I don't disagree with putting the three subheadlines article ID within `specialAssets` I think it would be better to retain the original name otherwise people may be confused as to why this article ID is deemed a "special asset".

Previous name - `newsThreeSubheadlines`

Reply:
agreed James sounds reasonable.